conf/machine/beaglev-starlight-jh7100.conf

```diff
@@ -28,7 +28,7 @@ RISCV_SBI_FDT ?= "jh7100-starlight.dtb"
 
 SERIAL_CONSOLES = "115200;ttyS0"
 
-MACHINE_EXTRA_RRECOMMENDS += " kernel-modules"
+MACHINE_EXTRA_RRECOMMENDS += " kernel-modules linux-firmware-beaglev-bcm43430"
```

Comment:
Where does this come from?

Reply:
hmm this comes from linux-starfive-dev recipe hmmm. We should switch to it as default I think

Reply:
Yeah, we probably should. Otherwise we should drop this as it fails to build by default.